### PR TITLE
doc: add demo for webserver workload

### PR DESCRIPTION
**Pull request by yangsoon**
add a application use **webserver** workload

### CODE REVIEW

File: docs/en/cue/workload-type.md [hunk 1]
```diff
@@ -135,87 +135,131 @@ metadata:
...
+Then, execute the following command
+```shell
+$ kubectl get deployment
+NAME             READY   UP-TO-DATE   AVAILABLE   AGE
+hello-world-v1   1/1     1            1           15s
+
+$ kubectl get svc
+NAME                           TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
+hello-world-trait-7bdcff98f7   ClusterIP   10.96.110.130   <none>        8000/TCP   32s
+kubernetes                     ClusterIP   10.96.0.1       <none>        443/TCP    5d18h
```
Comment (wonderflow) [inline docs/en/cue/workload-type.md:262]: delete it

File: docs/en/cue/workload-type.md [hunk 2]
```diff
@@ -135,87 +135,131 @@ metadata:
...
+Please save the Application example as file `app.yaml`, then create the new Application.
+```shell
+kubectl apply -f app.yaml
+```
+Wait for a while until the status of Application is `running`.
+```shell
+$ kubectl get application webserver-demo -o jsonpath='{.status.status}'
+running
+```
+Then, execute the following command
```
Comment (wonderflow) [inline docs/en/cue/workload-type.md:253]:
```suggestion
In the K8s cluster, you will see the following resources are created:
```

File: docs/en/cue/workload-type.md [hunk 3]
```diff
@@ -135,87 +135,131 @@ metadata:
...
+        cpu: "100m"
+```
+
+Please save the Application example as file `app.yaml`, then create the new Application.
+```shell
+kubectl apply -f app.yaml
+```
+Wait for a while until the status of Application is `running`.
+```shell
+$ kubectl get application webserver-demo -o jsonpath='{.status.status}'
```
Comment (wonderflow) [inline docs/en/cue/workload-type.md:250]: get the full yaml out

Comment (yangsoon) [reply docs/en/cue/workload-type.md]: Got it.

File: docs/en/cue/workload-type.md (hunk 2 above)
Comment (yangsoon) [reply docs/en/cue/workload-type.md]: This statement has been modified

File: docs/en/cue/workload-type.md (hunk 1 above)
Comment (yangsoon) [reply docs/en/cue/workload-type.md]: Got it.

**Review by wonderflow — APPROVED** [review]
LGTM

**Pull request merged**